website/docs/guides/orchestration/set-up-ci/more-complex-approach.md

```diff
@@ -0,0 +1,55 @@
+
+## Create a `release` branch in your git repo
+
+As noted above, this branch will outlive any individual feature, and will be the base of all feature development for a period of time. Your team might choose to create a new branch for each sprint (`release/sprint-01`, `release/sprint-02`, etc), tie it to a version of your data product (`release/1.0`, `release/1.1`), or just have a single `release` branch which remains active indefinitely.
+
+## Update your Development environment to use the `release` branch
+
+See [Custom branch behavior](/docs/dbt-cloud-environments#custom-branch-behavior). Setting `release` as your custom branch ensures that the IDE creates new branches and PRs with the correct target, instead of using `main`.
+
+## Create a new Release environment
+
+See [Create a new environment](/docs/collaborate/environments/dbt-cloud-environments#create-a-deployment-environment). The environment should be called **Release**. Just like your existing Production and Staging environments, it will be a Deployment-type environment.
+
+Set its branch to `release` as well.
+
+## Create a new job
+
+Use the **Continuous Integration Job** template, and call the job **Release CI Check**.
+
+In the Execution Settings, your command will be preset to `dbt build --select state:modified+`. Let's break this down:
+
+- [`dbt build`](/reference/commands/build) runs all nodes (seeds, models, snapshots, tests) at once in DAG order. If something fails, nodes that depend on it will be skipped.
+- The [`state:modified+` selector](/reference/node-selection/methods#the-state-method) means that only modified nodes and their children will be run ("Slim CI"). In addition to [not wasting time](https://discourse.getdbt.com/t/how-we-sped-up-our-ci-runs-by-10x-using-slim-ci/2603) building and testing nodes that weren't changed in the first place, this significantly reduces compute costs.
+
+To be able to find modified nodes, dbt needs to have something to compare against. Normally, we use the Production environment as the source of truth, but in this case there will be new code merged into `release` long before it hits the `main` branch and Production environment. Because of this, we'll want to defer the Release environment to itself.
+
+### Optional: also add a compile-only job
+
+Even when deferring to its own environment, dbt Cloud uses the last successful run of any job in that environment as its [comparison state](/reference/node-selection/syntax#about-node-selection). If you have a lot of PRs in flight, the comparison state could switch around regularly.
+
+Adding a regularly-scheduled job inside of the Release environment whose only command is `dbt compile` can regenerate a more stable manifest for comparison purposes.
```

Comment:
It's not possible to just defer to this job, right? It's _always_ environment-wide now?

Reply:
honest reaction is we should eliminate the last page. if we have to start a page with a 'YAGNI' flag i think we should just axe it.

Reply:
I wish 🥺 this is one of those "if you do have to do it, here's how to do it well" sections for large orgs who have an established process they want to migrate. I just want to dissuade people from thinking that they should work through _all_ these steps in order.